atlas-spec/mpc-engine/examples/run_mpc.rs

```diff
@@ -42,8 +33,10 @@ fn main() {
             .expect("every party should have a channel configuration");
         let c = circuit.clone();
         let party_join_handle = thread::spawn(move || {
-            let bytes = vec![0, 1, 2, 3];
-            let rng = mpc_engine::utils::rand::Randomness::new(bytes);
+            let mut rng = rand::thread_rng();
+            let mut bytes = [0u8; 500];
```

Comment:
No need to convert this into a vector when you make it a vector from the start.

```suggestion
            let mut bytes = vec![0u8; 500];
```